app/src/main/java/org/p2p/wallet/moonpay/ui/new/NewBuyPresenter.kt

```diff
@@ -204,7 +205,10 @@ class NewBuyPresenter(
 
     private fun isValidCurrencyForPay(): Boolean {
         val selectedCurrencyCode = selectedCurrency.code
-        if (selectedPaymentMethod.method == PaymentMethod.MethodType.BANK_TRANSFER) {
+        if (selectedPaymentMethod == null) {
```

Comment:
you can do it this check with local variable to remove `?:` later

Reply:
Yeah, but i don't understand with which value you want use ?: